src/locales/cn.json

```diff
@@ -206,14 +206,14 @@
 
     "list": {
       "title": "Patron列表",
-      "subtitle": "Here are some of the people or companies who support the project",
+      "subtitle": "这里有一些支持这个项目的人和公司",
       "patron": "Patron",
-      "amount": "Donation amount",
-      "amount-info": "The ordering is done via the value in U$D.",
-      "from": "From",
-      "not-possible": "It was not possible to load the patron list.",
-      "list-name": "Would you like to have your name or company listed here after a donation?",
-      "contact": "Contact me :)"
+      "amount": "捐赠数额",
+      "amount-info": "购买（捐赠）是通过美金完成的。",
```

Comment:
In this case, "ordering" means "sorting". The list us sorted/ordered by the converted donation value in dollars.
Maybe I need to change the info text in English.

Reply:
My bad. Should've checked the context while I felt confused.